geoips/commandline/log_setup.py

```diff
@@ -16,6 +16,24 @@
 import sys
 
 
+def log_with_emphasis(LOG, messages):
+    """Log the given message with emphasis provided a certain log type.
+
+    Parameters
+    ----------
+    LOG: logging function
+        Can be of any type [debug, info, interactive...]
+    message: str
+        The message to be logged with emphasis
+    """
+    max_message_len = min(80, max([len(message) for message in messages]))
```

Comment:
So it will be a max of 80, and shorter if the message is shorter?  That sounds good to me.

Reply:
`os.get_terminal_size()` could be used to retrieve the shape of the current terminal.

Reply:
I think we are way overthinking this.  If someone passes in a long line, just print it long.

If someone cares about how long their lines are, they can just adjust themselves.  I wouldn't worry at all about wrapping the individual lines.

If the line extends beyond the 80 characters, the ***** will stil just be 80 characters, and the line will extend beyond the ****s

Reply:
So the intended output would be:

```
********************************************************************************
*** Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. ***
********************************************************************************
```

This wraps like this on a small terminal:

![image](https://github.com/NRLMMD-GEOIPS/geoips/assets/9918036/5bc32d55-177c-4bf3-99b5-a98b383edb98)

and this on a larger terminal:

![image](https://github.com/NRLMMD-GEOIPS/geoips/assets/9918036/2cf1fa2c-cd28-4fce-a0fe-ccadf01ca2fc)

And with a smaller log message, the intended output would be:

```
********************************************************************************
*** Testing string. ***
********************************************************************************
```

This wraps like this on a small terminal:

![image](https://github.com/NRLMMD-GEOIPS/geoips/assets/9918036/96d0e320-389b-41fc-bbdb-62616cd97e15)

and this on a larger terminal:

![image](https://github.com/NRLMMD-GEOIPS/geoips/assets/9918036/a69a170c-37f0-42c6-a1f6-110d9bbfe43a)

Is that right?

Reply:
My mistake - I forgot it'll shorten the `*`s if the message is shorter.